src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -638,6 +638,11 @@ private PSModuleInfo ImportModule_LocallyViaName(ImportModuleOptions importModul
 
                             if (!found)
                             {
+                                // If the path ends with a directory separator, remove it
```

Comment:
By looking at the code in the file, I get a feeling that it is better to place this new change (that removes trailing separator) before the call to `LoadUsingMultiVersionModuleBase` that is earlier in the file: 
`foundModule = LoadUsingMultiVersionModuleBase(rootedPath,`
This way version selection in the multi-version case will still work even with trailing separator in module path.

Reply:
Good catch.  I'll move it up.